### DISCUSSION TITLE
Customizing selective elements in grapesjs react.

### DISCUSSION THREAD

**Discussion by FahemAhmad** [GrapesJS/grapesjs#5374] [Q&A] [answered]
hi @artf . I was following this grapes js react example. You are making a custom grapes js style manager here. Is there a way i can just customise selective components, for example i keep the original color picker or slider, but redesign the select element.

https://stackblitz.com/edit/grapesjs-react-custom-ui?file=src%2FApp.tsx,src%2Fcomponents%2FRightSidebar.tsx,src%2Fcomponents%2FCustomStyleManager.tsx,src%2Fcomponents%2FStylePropertyField.tsx

the customisation is in stylePropertyField.tsx

**Comment by artf** [comment, marked as answer]
No, being a custom UI for the Style Manager you have to provide the UI for each type
